core/build.gradle

```diff
@@ -52,15 +52,15 @@ android {
             dimension "market"
         }
     }
-
 }
 
 dependencies {
-    implementation "androidx.appcompat:appcompat:1.1.0"
-    implementation "androidx.preference:preference:1.1.0"
-    annotationProcessor "androidx.annotation:annotation:1.1.0"
+    annotationProcessor 'androidx.annotation:annotation:1.1.0'
+    implementation 'androidx.appcompat:appcompat:1.1.0'
+    implementation 'androidx.media:media:1.1.0'
+    implementation 'androidx.preference:preference:1.1.1'
```

Comment:
This had to be updated in both `app` and `core`. Could you please extract variables for the androidx versions, so they are consistent?